LICENSES/vendor/github.com/russross/blackfriday/v2/LICENSE

```diff
@@ -0,0 +1,33 @@
+= vendor/github.com/russross/blackfriday/v2 licensed under: =
+
+Blackfriday is distributed under the Simplified BSD License:
```

Comment:
for reference, this is the 2-clause BSD license (https://github.com/cncf/foundation/blob/master/whitelist-policy.md#approved-licenses-for-whitelist)

Reply:
Sorry for not good at dependency area, I'll take a deeper look at it and try to reply your comments tomorrow(my timezone).

Reply:
no change was required here, I was just noting the license was acceptable